### DISCUSSION TITLE
Is there a way to load pdf from a server?

### DISCUSSION THREAD

**Discussion by Rmn97** [stephanrauh/ngx-extended-pdf-viewer#1299] [Q&A] [answered]
Hi, I'm trying setting up my angular app with this library and I have a service which gets pdf files from a server using a get method and I notice that the library renders local pdf files from assets folder or any local folder.
How can I change that? Is there an alternative?

**Comment by stephanrauh** [comment, marked as answer]
You can pass the PDF file as a Blob (see https://pdfviewer.net/extended-pdf-viewer/blob) or as a Base64 encoded string (see https://pdfviewer.net/extended-pdf-viewer/base64). The two pages also contain example code.

At the moment, Blob are sometimes a bit slow on my Laptop, but I'm not sure whether this is a side-effect of something completely different (such as an adblocker). If you observe the same problem, please tell me!

**Reply by Rmn97** [reply]
Okay I see, do I still need this set up in angular.json right? : 
```json
 "assets": [
    "src/favicon.ico",
    "src/assets",
    {
      "glob": "**/*",
      "input": "node_modules/ngx-extended-pdf-viewer/assets/",
      "output": "/assets/"
    }
```
and then follow the code in https://pdfviewer.net/extended-pdf-viewer/blob ?

**Reply by stephanrauh** [reply]
Yes, that's correct.

**Reply by Rmn97** [reply]
Thank you so much for your answer, let me see what I can do with my api url
